issue-4-functions-in-dart/issue-4-functions-in-dart.md

```diff
@@ -200,6 +203,8 @@ void main(List<String> args) {
 }
 ```
 
+Note: we are not actually invoking the foo function, rather printing its descriptor!
```

Comment:
To be more precise we print `foo`'s tear-off - which is just a closure forwarding invocations to `foo`. 

Compare:

```dart
void foo() { }

void main() {
  var f = foo; // foo's closurisation also known as tear-off.
  print(f); // closures are just normal objects, so you can print them

  var f_toString = f.toString;  // you can also tear off methods on objects
                                // you get a closure which invokes method on a fixed object
  print(f_toString());  // the same as calling f.toString() here.
}
```